File: lab/python/congruentes/congruentes.py
def congruentes(n, j, m):
    alvo = j % m  # resto desejado
    contador = 0
    numero = 0

    while contador < n:
        if numero % m == alvo:
            print(numero)
            contador += 1
        numero += 1


congruentes(5, 35, 39)

File: lab/javascript/congruentes/congruentes.js
function congruentes(n, j, m){
    let alvo = j % m;  // resto desejado
    let contador = 0;
    let numero = 0;

    while (contador < n){
        if (numero % m == alvo){
            console.log(numero);
            contador ++;
        }
        numero ++;
    }
}

congruentes(5, 35, 39)
